rmw_fastrtps_cpp/src/rmw_init.cpp

```diff
@@ -93,13 +93,13 @@ rmw_init(const rmw_init_options_t * options, rmw_context_t * context)
     return RMW_RET_INVALID_ARGUMENT;
   }
 
-  const rmw_context_t zero_context = rmw_get_zero_initialized_context();
-  assert(0 == std::memcmp(context, &zero_context, sizeof(rmw_context_t)));
```

Comment:
@ivanpauno also unrelated, and can be backtracked to the discussion that took place about UB in `rmw_init()`.

Reply:
See https://github.com/ros2/rmw_connext/pull/443#discussion_r458314262